.evergreen/config.yml

```diff
@@ -1383,7 +1383,6 @@ tasks:
         type: setup
         params:
           updates:
-            - {key: NPM_VERSION, value: '9'}
```

Comment:
Removed all these just to show in the majority of our tests the fallback script works for all versions.